### ISSUE TITLE
Show runtime errors

### ISSUE THREAD

**Issue by jpoehnelt** [google/playground-elements#259]
Would it be possible to intercept runtime errors and have another component to display these?

**Comment by aomarks** [comment]
Yes, that should be possible, and something like a `<playground-error>` component makes sense!

We could something like this in `<playground-preview>`:

```js
iframe.contentWindow.addEventListener('error', (e) => {
  this._project.reportPreviewError(e);
  // Without capture we get JS errors. With capture we
  // also get loading errors, like a 404ing img src.
}, {capture: true});
```

And then `<playground-project>` could emit a `preview-error` event.

And then a new `<playground-errors>` component could listen for `preview-error` events from its `this._project.
